src/styles/index.scss

```diff
@@ -1,3 +1,157 @@
+@use 'variables';
+
+// #region fonts
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.roboto-bold {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+// #endregion
+
 body {
   margin: 0;
 }
+
+.stars {
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-size: 80%;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $mainAccentColor;
+
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding-inline: 16px;
+  padding-block: 32px 16px;
+
+  border: 1px solid #f3f3f3;
+
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin-inline: 3px;
+  }
+
+  &__name {
+    margin-top: 40px;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondaryColor;
+  }
+
+  // #region review
+  &__review {
+    margin-top: 16px;
+
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review-stars {
+    width: 96px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__review-text {
+    width: 56px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  // #endregion
+
+  // #region price
+  &__price {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+    text-align: left;
+    color: $secondaryColor;
+  }
+
+  &__price-numbers {
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    text-align: right;
+  }
+
+  // #endregion
+
+  &__buy {
+    box-sizing: border-box;
+    display: block;
+    margin-top: 16px;
+    width: 166px;
+    height: $buyButtonHeight;
+
+    line-height: $buyButtonHeight;
+    font-size: 14;
```

Comment:
The font-size property on line 142 is missing a unit. It should be specified as '14px' or '14rem', etc., to ensure proper rendering across different browsers.